apps/graph-api/src/guards/read-only-deployment-guard.service.ts

```diff
@@ -2,7 +2,7 @@ import { ConfigService } from '#graph-lib/config';
 import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
 
 @Injectable()
-export class ReadOnlyGuard implements CanActivate {
+export class ReadOnlyDeploymentGuard implements CanActivate {
```

Comment:
We have this Guard duplicated in other applications but I didn't want to increase the size of this PR but those will be also replaced and refactored in the future.

Reply:
Maybe we can add it to some existing story as a comment

Reply:
We'll refactor this to `libs/utils`, but we can't until after we deal with the `ConfigService` dependency.